Makefile

```diff
@@ -1,63 +1,139 @@
+#
+# Makefile for the Links Programming Language.
+#
+
+# This file contains instructions for building a bleeding-edge version
+# of Links in release or development mode with and without database
+# support.
+
+# LATEST STABLE RELEASE
+# =====================
+# To install the latest stable release of Links please use opam, e.g.
+# $ opam install links
+# .. and to enable database support
+# $ opam install links-postgresql
+
+# To build the bleeding-edge version of Links please refer to the
+# instructions below.
+
+# DEVELOPMENT
+# ===========
+# By default the build rule `all' attempts to build the latest
+# development version of Links with database support, thus the
+# following command suffices to build everything
+# $ make all
+# For legacy reasons the rule `nc' (native compile) is an alias of
```

Comment:
is there anything preventing us from just getting rid of the `nc` target?

Reply:
Well, I often type `make nc`. It seems to be imprinted in my muscle memory.

Reply:
time to break with this habit then? ;)